build-scripts/scriptUtils.js

```diff
@@ -0,0 +1,23 @@
+/*!
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: MIT
+ */
+
+const path = require('path')
+
+repoRoot = path.dirname(__dirname)
+tasksDirectory = 'Tasks'
+inTasks = path.join(repoRoot, tasksDirectory)
+outTasks = path.join(repoRoot, '_build', tasksDirectory)
+outBuildTasks = path.join(repoRoot, '_build', tasksDirectory)
```

Comment:
outTasks and outBuildTasks  are the same value

Reply:
Yes they do, but I like having both of them because they are used in different ways. outTasks is used as an output and outBuildTasks is used as an input.

Reply:
Knowing nothing about the context here, if you are going to keep both suggest only evaluating the expression once and just assigning one variable to the other. It'll make it more obvious that they're intentionally the same.

If we are going to keep both `outBuidTasks` is a very confusing name for an input variable.

Reply:
I renamed all of the variables, they make way more sense now